templates/settings.yaml.epp

```diff
@@ -3,7 +3,7 @@
   Optional[Boolean] $test,
   Optional[Stdlib::HTTPUrl] $relay_api_url,
   Optional[Sensitive[String]] $relay_connection_token,
-  Optional[Variant[Array[Sensitive[String]], Sensitive[String]]] $relay_trigger_token,
+  Optional[Variant[Sensitive[Array[String]], Sensitive[String]]] $relay_trigger_token,
```

Comment:
If `Array[Sensitive[String]]` is impossible, we can simplify the few lines below this to `$relay_trigger_token.then |$ts| { $ts.unwrap }` instead of having the complicated type check.

Reply:
`Array[Sensitive[String]] ` _is_ possible to pass from code, but it's just not (easily) possible using Hiera, so for defensive reasons we might want to just keep this code?

Reply:
If it no longer matches the type definition of the variable, I don't see any reason to keep it.

Reply:
@impl Ok, I simplified the unwrapping